--- Engine/Engine.pro.orig	2015-03-21 18:34:16.000000000 +0100
+++ Engine/Engine.pro	2015-03-21 18:37:24.000000000 +0100
@@ -360,8 +360,8 @@
                 $$BREAKPAD_PATH/common/mac/MachIPC.mm
 }
 
-# other *nix
-unix:!mac {
+# linux 
+linux {
         SOURCES += $$BREAKPAD_PATH/client/linux/handler/exception_handler.cc \
                 $$BREAKPAD_PATH/client/linux/handler/minidump_descriptor.cc \
                 $$BREAKPAD_PATH/client/linux/crash_generation/crash_generation_client.cc \
--- Gui/KnobGuiTypes.cpp.orig	2015-03-21 19:14:52.000000000 +0100
+++ Gui/KnobGuiTypes.cpp	2015-03-21 19:15:52.000000000 +0100
@@ -1553,7 +1553,7 @@
 void
 Choice_KnobGui::onCurrentIndexChanged(int i)
 {
-    pushUndoCommand( new KnobUndoCommand<int>(this,_knob->getValue(0,false),i, false) );
+    pushUndoCommand( new KnobUndoCommand<int>(this,_knob->getValue(0,false),i, 0, false, 0) );
 }
 
 void
--- Project.pro.orig	2015-03-21 19:40:19.000000000 +0100
+++ Project.pro	2015-03-21 20:05:40.000000000 +0100
@@ -14,8 +14,9 @@
     Gui \
     Renderer \
     Tests \
-    App \
-    CrashReporter
+    App
+
+win32|mac|linux {SUBDIRS+=CrashReporter}
 
 OTHER_FILES += \
     Global/Enums.h \
