--- Gui/Gui.cpp.orig	2014-08-18 02:00:24.358000172 +0200
+++ Gui/Gui.cpp	2014-08-18 02:04:22.131999690 +0200
@@ -805,7 +805,7 @@
     QObject::connect(_imp->actionSave_project, SIGNAL(triggered()), this, SLOT(saveProject()));
     _imp->actionSaveAs_project = new QAction(this);
     _imp->actionSaveAs_project->setObjectName(QString::fromUtf8("actionSaveAs_project"));
-    _imp->actionSaveAs_project->setShortcut(QKeySequence::SaveAs);
+    _imp->actionSaveAs_project->setShortcut(QKeySequence(Qt::CTRL+Qt::SHIFT+Qt::Key_S));
     _imp->actionSaveAs_project->setIcon(get_icon("document-save-as"));
     QObject::connect(_imp->actionSaveAs_project, SIGNAL(triggered()), this, SLOT(saveProjectAs()));
     _imp->actionPreferences = new QAction(this);
@@ -815,7 +815,7 @@
     _imp->actionExit = new QAction(this);
     _imp->actionExit->setObjectName(QString::fromUtf8("actionExit"));
     _imp->actionExit->setMenuRole(QAction::QuitRole);
-    _imp->actionExit->setShortcut(QKeySequence::Quit);
+    _imp->actionExit->setShortcut(QKeySequence(Qt::CTRL+Qt::Key_Q));
     _imp->actionExit->setShortcutContext(Qt::WindowShortcut);
     _imp->actionExit->setIcon(get_icon("application-exit"));
     _imp->actionProject_settings = new QAction(this);
