--- CMakeLists.txt.orig	2014-06-25 07:46:16.753472660 +0200
+++ CMakeLists.txt	2014-06-25 07:47:04.951404727 +0200
@@ -118,7 +118,7 @@
 set (USE_OPENJPEG ON CACHE BOOL "Use OpenJpeg if found")
 set (USE_OCIO ON CACHE BOOL "Use OpenColorIO for color management if found")
 set (USE_OPENCV ON CACHE BOOL "Use OpenCV if found")
-set (USE_OPENSSL ON CACHE BOOL "Use OpenSSL if found")
+#set (USE_OPENSSL ON CACHE BOOL "Use OpenSSL if found")
 set (USE_FREETYPE ON CACHE BOOL "Use Freetype if found")
 set (USE_GIF ON CACHE BOOL "Use GIF if found")
 set (USE_LIBRAW ON CACHE BOOL "Use LibRaw if found")
